makelib/k8s_tools.mk

```diff
@@ -135,7 +135,7 @@ $(KIND):
 # kubectl download and install
 $(KUBECTL):
 	@$(INFO) installing kubectl $(KUBECTL_VERSION)
-	@curl -fsSLo $(KUBECTL) --create-dirs https://storage.googleapis.com/kubernetes-release/release/$(KUBECTL_VERSION)/bin/$(HOSTOS)/$(SAFEHOSTARCH)/kubectl || $(FAIL)
+	@curl -fsSLo $(KUBECTL) --create-dirs https://dl.k8s.io/$(KUBECTL_VERSION)/bin/$(HOSTOS)/$(SAFEHOSTARCH)/kubectl || $(FAIL)
```

Comment:
the [official docs](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) seem to still have the `/release/` part of the URL, e.g. `https://dl.k8s.io/release/v1.32.0/bin/linux/amd64/kubectl`.  Do you want to keep that part in this change?

Reply:
You are right, copied the deeplink url from here :)
https://kubernetes.io/releases/download/#binaries

Changed it to path `/release`


Thx for quick review!